utility/tor/tor_launcher_impl.cc

```diff
@@ -74,22 +74,6 @@ static void TearDownPipeHack() {
 
 namespace tor {
 
-#if defined(OS_POSIX) && !defined(OS_MACOSX)
```

Comment:
@darkdh I think you are aware fully but just flagging in case.

Reply:
addressed in 5327a2c4e10ba80a9f072893186e30ecd941c6cf